hq_superset/tasks.py

```diff
@@ -16,34 +13,3 @@ def refresh_hq_datasource_task(domain, datasource_id, display_name, export_path,
         AsyncImportHelper(domain, datasource_id).mark_as_complete()
         raise
     os.remove(export_path)
-
```

Comment:
From the commit message:
> so that it can get the OAuth token to authenticate with HQ.

Speaking out of inexperience in this area, but can't we simply pass the token to the celery task?

Reply:
(Just closing the loop here. It needs the request session. We could serialize it and pass it to the task, but something about that feels awkward to me. I'm happy to be educated though. :) )